std_msgs/Header header

# x y z roll pitch yaw
float64[6] end_pose

# joint position information
float64[] joint_position

# joint velocity information
float64[] joint_velocity

# joint effort information
float64[] joint_effort